### DISCUSSION TITLE
Download bug

### DISCUSSION THREAD

**Discussion by Heruar** [aclap-dev/video-downloadhelper#628] [Help and Q&A]
### Your question

I couldn't download the video on behance, below is the bug information that popped up.
{
  "version": "9.0.1.18",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.behance.net/gallery/186524313/MERCEDES-BENZ-SHAPED-BY-DESIRE?tracking_source=curated_galleries_3d-art",
  "error": {
    "details": "SideDownload error: : [in#0 @ 00000244f647ac00] Error opening input: Server returned 403 Forbidden (access denied)\r\nError opening input file https://vod-adaptive-ak.vimeocdn.com/exp=1716610284~acl=%2F7c6cec38-6469-4a45-8567-2835078f4fe1%2F%2A~hmac=1dd3d37ab26f729d685b43d966b23414ad52397b251799eb8f7fa8f2398be535/7c6cec38-6469-4a45-8567-2835078f4fe1/sep/video/b79737ea/playlist.m3u8?query_string_ranges=1.\r\nError opening input files: Server returned 403 Forbidden (access denied)\r\n",
    "downloadable_id": "downloadable_1103210634032293",
    "error": "coapp_failure",
    "id": "coapp_failure_568c2a91-5751-4228-8f11-ea8f14d6598c"
  }
}

### Addon & CoApp version and details

version: 9.0.1.18
target: google
channel: stable
lang: zh-CN
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"47383c8921f845ad"}
platform: x86-32 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "C:\\Users\\设计\\dwhelper",
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 4
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3593, Firefox 126.0 64-bit, licensed VDH 9.0.1.22a1 beta, CoApp 2.0.19.

VDH downloaded the Mercedes ad fine for me.  Took about 7 seconds.  VLC played the whole 1:30 just fine.  This would not be the first time Firefox did a better job with something that Chrome.

**Reply by Heruar** [reply]
It's nerve-racking. I have to repurchase the license for Firefox.
